src/Web/Controllers/OrderController.cs

```diff
@@ -181,6 +183,24 @@ public async Task<IActionResult> ChargebackOrder(OrderViewModel viewModel, Charg
                 Processor = "Fraud Protection sample site",
             };
 
+            if(!String.IsNullOrEmpty(effectiveStartDate) )
+            {
+                DateTime labelEffectiveStartDate;
+                if (DateTime.TryParse(effectiveStartDate, out labelEffectiveStartDate))
+                    {
+                    label.EffectiveStartDate = labelEffectiveStartDate;
```

Comment:
Can you confirm that when sample site sends the Label API request that these dates have the timezone offset included?

Reply:
Yes, they do:
"EffectiveStartDate": "2021-06-03T14:34:59-07:00",
  "EffectiveEndDate": "2022-06-03T14:34:59-07:00",